From 89a3ae2de21e528b8feb80dbea3928d43982d3be Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@openwide.fr>
Date: Thu, 21 Aug 2014 18:18:16 +0200
Subject: [PATCH 1/1] rename log_msg to libinput_log_msg

This fixes a conflict between libevdev and libinput on the definition
of the log_msg() symbol.

http://autobuild.buildroot.net/results/c13/c133b7c706ee31302125df8ca94f4d0f0152c6c6/build-end.log

Signed-off-by: Romain Naour <romain.naour@openwide.fr>
[yann.morin.1998@free.fr: rebase on-top of 0.6.0]
Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
---
 src/libinput-private.h | 14 +++++++-------
 src/libinput.c         |  2 +-
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/libinput-private.h b/src/libinput-private.h
index f9d2f87..ac1e1c4 100644
--- a/src/libinput-private.h
+++ b/src/libinput-private.h
@@ -116,15 +116,15 @@ typedef void (*libinput_source_dispatch_t)(void *data);
 typedef void (*libinput_source_dispatch_t)(void *data);
 
 
-#define log_debug(li_, ...) log_msg((li_), LIBINPUT_LOG_PRIORITY_DEBUG, __VA_ARGS__)
-#define log_info(li_, ...) log_msg((li_), LIBINPUT_LOG_PRIORITY_INFO, __VA_ARGS__)
-#define log_error(li_, ...) log_msg((li_), LIBINPUT_LOG_PRIORITY_ERROR, __VA_ARGS__)
-#define log_bug_kernel(li_, ...) log_msg((li_), LIBINPUT_LOG_PRIORITY_ERROR, "kernel bug: " __VA_ARGS__)
-#define log_bug_libinput(li_, ...) log_msg((li_), LIBINPUT_LOG_PRIORITY_ERROR, "libinput bug: " __VA_ARGS__);
-#define log_bug_client(li_, ...) log_msg((li_), LIBINPUT_LOG_PRIORITY_ERROR, "client bug: " __VA_ARGS__);
+#define log_debug(li_, ...) libinput_log_msg((li_), LIBINPUT_LOG_PRIORITY_DEBUG, __VA_ARGS__)
+#define log_info(li_, ...) libinput_log_msg((li_), LIBINPUT_LOG_PRIORITY_INFO, __VA_ARGS__)
+#define log_error(li_, ...) libinput_log_msg((li_), LIBINPUT_LOG_PRIORITY_ERROR, __VA_ARGS__)
+#define log_bug_kernel(li_, ...) libinput_log_msg((li_), LIBINPUT_LOG_PRIORITY_ERROR, "kernel bug: " __VA_ARGS__)
+#define log_bug_libinput(li_, ...) libinput_log_msg((li_), LIBINPUT_LOG_PRIORITY_ERROR, "libinput bug: " __VA_ARGS__);
+#define log_bug_client(li_, ...) libinput_log_msg((li_), LIBINPUT_LOG_PRIORITY_ERROR, "client bug: " __VA_ARGS__);
 
 void
-log_msg(struct libinput *libinput,
+libinput_log_msg(struct libinput *libinput,
 	enum libinput_log_priority priority,
 	const char *format, ...);
 
 void
-log_msg(struct libinput *libinput,
+libinput_log_msg(struct libinput *libinput,
 	enum libinput_log_priority priority,
 	const char *format, ...);
 
diff --git a/src/libinput.c b/src/libinput.c
index eec9efb..4e3f6fa 100644
--- a/src/libinput.c
+++ b/src/libinput.c
@@ -110,7 +110,7 @@ static struct log_data log_data = {
 }
 
 void
-log_msg(struct libinput *libinput,
+libinput_log_msg(struct libinput *libinput,
 	enum libinput_log_priority priority,
 	const char *format, ...)
 {
-- 
1.9.3

